src/MobileBookingHeader/MobileBookingDetail.js

```diff
@@ -48,14 +43,17 @@ type MobileBookingControlsProps = {|
 
 const MobileBookingControls = (props: MobileBookingControlsProps) => (
   <React.Fragment>
-    <a
-      rel="noopener noreferrer"
-      target="_blank"
-      href={props.manageBookingURL}
-      className="manageBookingButton"
-    >
-      Manage My Booking
-    </a>
+    <div className="manageBookingButton">
+      <Button
+        block
+        onClick={() => {
+          window.open(props.manageBookingURL, '_blank');
```

Comment:
Should also add `window.opener=null` :slightly_smiling_face:

Reply:
Isn't it considered an unsafe practice not to do it when `_blank`ing? One can modify the `opener` of the new tab, ie kiwi.com+smart-faq.

Reply:
To my knowledge it is unsafe when it's user generated content. If someone can control the site that is opened, they'll have access to the opener `window` and do all kinds of nasty stuff. AFAIK this is not a risk in our case as the link is also on kiwi.com, so it doesn't add a new security risk.

However we can add it just in case :man_shrugging:

Reply:
ie. `window.open(props.manageBookingURL, '_blank').opener = null;`

Reply:
Agree with you it's not a risky case. I was curious as to why have it before and remove it now tho. Also it's quite cheap to add I'd say so why not? :smile:

Reply:
Ok I added it :-) I chose this implementation: https://stackoverflow.com/a/46958731